mig/install/MiGserver-template.conf

```diff
@@ -196,6 +196,18 @@ gdp_email_notify = __GDP_EMAIL_NOTIFY__
 # NOTE: the sftpsubsys service is advertized as just sftp to fit the protocol.
 storage_protocols = __STORAGE_PROTOCOLS__
 
+# Optional limit which instructs the web backends to deliver at most N bytes of
+# output to the users in order to avoid excessive memory use when serving files
+# which are subject to in-memory buffering. Defaults to a value of -1 which
+# disables the limit completely while any other positive value is intepreted
+# as a total number of bytes to allow serving in a single request. The
```

Comment:
The last bit of this sentence ends rather abruptly here and in the other copies as discussed on direct chat.